source/style/content.css

```diff
@@ -187,3 +187,23 @@ code.refined-twitter_markdown {
 .DashUserDropdown .dropdown-caret {
 	transform: translateX(-145px);
 }
+
+/* Plugin-generated div that display the alternative text of an image */
+.refined-twitter_image-alt {
+	background-color: rgba(var(--refined-twitter_bgcolor-values), 0.1);
```

Comment:
I think we should choose a hardcoded background color that matches either the light or dark theme, not the user's custom color.